worlds/ff4fe/__init__.py

```diff
@@ -191,6 +214,11 @@ def set_rules(self) -> None:
                         continue
                 add_item_rule(self.get_location(location.name),
                               lambda item: item.name not in items.characters)
+                if self.options.ItemPlacement.current_key == "major_minor_split" and location.major_slot == False:
+                    add_item_rule(self.get_location(location.name),
+                                  lambda item: (item.classification & ItemClassification.progression == 0) and item.name != "DkMatter")
```

Comment:
This line as is does not actually resolve the major-minor split and DkMatter in the item pool.

Instead, the code should be the following.
```python
add_item_rule(self.get_location(location.name),
                                  lambda item: (item.classification & ItemClassification.progression == 0) or item.name == "DkMatter")
```